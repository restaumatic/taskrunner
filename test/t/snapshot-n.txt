snapshot -n
echo "Always executed"
