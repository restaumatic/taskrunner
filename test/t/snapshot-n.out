-- output:
[toplevel] stdout | Always executed
